source/docs/a1.why_rust.md

```diff
@@ -34,4 +34,4 @@ and etc.
 
 Rust **doesn't use an automated garbage collection** system\(GC\) by default.
 
-Rust compiler observes the code **at compiler time** and helps to [**prevent many types of errors**](https://doc.rust-lang.org/error-index.html) that are possible to write in C++
+Rust compiler observes the code **at compile time** and helps [**preventing many types of errors**](https://doc.rust-lang.org/error-index.html) that are possible to write in C/C++, which makes it among the first contenders to Clang.
```

Comment:
> , which makes it among the first contenders to Clang
Let's keep this generic to all audience, not targeting only systems programmers :)